Lib/BingAds/BingAdsApiWrapper.php

```diff
@@ -187,4 +211,15 @@ protected function _getClientProxyWithAccountId($endPoint, $accountId, $apiVersi
 			= $this->_clientProxyFactory->createClientProxyWithAccountId($endPoint, $accountId, $apiVersion);
 	}
 
+	/**
+	 * Throw the real exception...
+	 * @param SoapFault $e
+	 *
+	 * @throws Exception
+	 */
+	protected function _throwExceptionBySoapFault(SoapFault $e) {
```

Comment:
Is het niet logischer om hier een SoapFaultException voor te maken die `$e` ontvangt?